src/utill.rs

```
@@ -234,47 +231,20 @@ pub fn setup_directory_logger(filter: LevelFilter) {
             .build(Root::builder().appender("stdout").build(filter))
             .unwrap();
 
-        log4rs::init_config(config).unwrap();
+        if let Err(e) = log4rs::init_config(config) {
+            eprintln!("Failed to initialize logger: {}", e);
+        }
```

Same.

---

Have removed all suppressing error now.